build/frpc/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.16 as builder
+FROM golang:1.18.1 as builder
```

Comment:
shall we use 1.19?

Reply:
Sure Jun

Reply:
Fixed in https://github.com/spectrocloud/frp/pull/2/commits/7d947f628fcabb3d771f5827f78069b1482fe7d2